audit-resolve.json

```diff
@@ -103,19 +104,19 @@
       "madeAt": 1622647824779
     },
     "1675|@mojaloop/central-services-shared>shins>sanitize-html": {
-        "decision": "ignore",
-        "madeAt": 1623336829070,
-        "expiresAt": 1623941956342
-      },
-      "1676|@mojaloop/central-services-shared>shins>sanitize-html": {
-        "decision": "ignore",
-        "madeAt": 1623336829070,
-        "expiresAt": 1623941956342
-      },
-      "1693|@mojaloop/central-services-shared>shins>sanitize-html>postcss": {
-        "decision": "ignore",
-        "madeAt": 1623336829070,
-        "expiresAt": 1623941956342
+      "decision": "ignore",
+      "madeAt": 1626148943210,
+      "expiresAt": 1628740931974
+    },
+    "1676|@mojaloop/central-services-shared>shins>sanitize-html": {
+      "decision": "ignore",
+      "madeAt": 1626148943210,
+      "expiresAt": 1628740931974
+    },
+    "1693|@mojaloop/central-services-shared>shins>sanitize-html>postcss": {
+      "decision": "ignore",
+      "madeAt": 1626148971400,
+      "expiresAt": 1628740965642
```

Comment:
We're trying to be a little more thorough about this... can you please comment on why they are being ignored?

e.g. no fix available, or perhaps only low risk

Reply:
no fix available.
one of them I remember was "fixable" but selecting fix didn't do any package changes.

Reply:
ok. For the next pr, please note the severity level of the ignored vulns - we're working on a more thorough approach, but if we are all a little more aware of it then that will help.

Reply:
where should that note live? in `audit-resolve.json`?